src/test/java/org/candlepin/subscriptions/security/IdentityHeaderAuthenticationDetailsServiceTest.java

```diff
@@ -20,16 +20,20 @@
  */
 package org.candlepin.subscriptions.security;
 
-import static org.hamcrest.MatcherAssert.*;
-import static org.junit.jupiter.api.Assertions.*;
-import static org.mockito.ArgumentMatchers.*;
-import static org.mockito.Mockito.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verifyZeroInteractions;
+import static org.mockito.Mockito.when;
```

Comment:
I'm not going to ask you to change it, but I like the static wildcard imports better.  Maybe we should figure out a team style to agree on for this.

Reply:
I like that as well, but the style guide -- and I think sonarcloud -- object.

Reply:
Google Style is what drives this import order/style.  It doesn't bother me enough to think its worth deviatiating from those out of the box rules, because I usually collapse the imports in my IDE anyway.  Willing to be convinced otherwise though